connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoDataSourceUtils.scala

```diff
@@ -172,6 +177,23 @@ object KustoDataSourceUtils {
     SinkParameters(writeOptions, sourceParameters)
   }
 
+  def getClientRequestProperties(parameters: Map[String, String]): Option[ClientRequestProperties] = {
```

Comment:
Maybe we should add this to the client as well,.
The method name will be: fromJsonString

Reply:
will keep this here until we have it there .. will add a TODO